buildSrc/src/main/kotlin/org/cqfn/save/generation/Generation.kt

```diff
@@ -128,11 +129,21 @@ fun FunSpec.Builder.generateOptions(jsonObject: Map<String, Option>): FunSpec.Bu
         // We replace whitespaces to `·`, in aim to avoid incorrect line breaking,
         // which could be done by kotlinpoet
         option += "description = \"${it.value.description.replace(" ", "·")}\"\n"
-        option += ").value\n"
+        option += ")\n"
         this.addStatement(option)
     }
     return this
 }
+
+// Assign class members to options
+fun FunSpec.Builder.assignMembersToOptions(jsonObject: Map<String, Option>): FunSpec.Builder {
+    jsonObject.forEach {
+        val assign = "this.${it.key} = ${it.key}\n"
```

Comment:
Is line terminator required here? I think blank lines between statements are not needed here

Reply:
Agree, removed